backend/app/routers/persistence.py

```
@@ -222,13 +250,147 @@ async def get_graph(graph_id: str, userId: str = Query(...)):
+    return SpecialNoteFileListResponse(
+        items=[to_special_note_file(row) for row in rows],
+        usedBytes=used_bytes,
+        limitBytes=app_settings.special_note_storage_quota_bytes,
+    )
+
+
+@router.post("/graphs/{graph_id}/special-notes", response_model=SpecialNoteFile, status_code=201)
+async def upload_special_note_file(
+    graph_id: str,
+    userId: str = Form(...),
+    file: UploadFile = File(...),
```

Ruff (0.16.1)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)